tests/e2e/ADDING_TESTS.md

```diff
@@ -68,7 +68,11 @@ const test = () => {
   Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(firstReportIDInList));
 
   // markEnd will be called in the Screen's implementation
-  performance.subscribeToMeasurements("navigateToReport", (measurement) => {
+  Performance.subscribeToMeasurements((measurement) => {
+      if (!measurement.name !== "navigateToReport") {
+        return;
+      }
+
```

Comment:
NAB: the comment on the follow line is extremely low value and can be removed.

Reply:
agree, removing now